samples/ext/nffs/src/main.c

```diff
@@ -0,0 +1,140 @@
+/*
+ * Copyright (c) 2016 Nordic Semiconductor ASA
```

Comment:
I belief this file is whole new - you can add your or your company as copyright owner.